be/src/storage/sstable/table.h

```diff
@@ -54,7 +51,7 @@ class Table {
     // Calls (*handle_result)(arg, ...) with the entry found after a call
     // to Seek(key).  May not make such a call if filter policy says
     // that key is not present.
-    Status MultiGet(const ReadOptions&, size_t n, const Slice* keys, const KeyIndexesInfo& keys_info,
+    Status MultiGet(const ReadOptions&, const Slice* keys, const std::set<KeyIndex>& key_indexes,
```

Comment:
think about if willing to treat the sstable as library, it is the caller to adapt to the library interface, not the library adapt to the caller's usage. will this keyIndexes generic enough for the library reuse in future?

Also the comments of the interface seems out of date, didn't know what on earth the comment is talking about.